bundles/binding/org.openhab.binding.modbus/src/main/java/org/openhab/binding/modbus/internal/ModbusBinding.java

```diff
@@ -183,7 +187,20 @@ protected void internalReceiveCommand(String itemName, Command command) {
             if (provider.providesBindingFor(itemName)) {
                 ModbusBindingConfig config = provider.getConfig(itemName);
                 ModbusSlave slave = modbusSlaves.get(config.slaveName);
-                slave.executeCommand(command, config);
+                /* we need to handle rollershutter commands in a different way */
+                if (config.getItemClass() == RollershutterItem.class) {
```

Comment:
I understand what you mean. However, there are very similar cases in Slave already, for example:
````
        } else if (command instanceof OnOffType) {
            newValue = new SimpleRegister();
            if (command.equals(OnOffType.ON)) {
                newValue.setValue(1);
            } else if (command.equals(OnOffType.OFF)) {
                newValue.setValue(0);
            }
````

I would keep the command handling in same place, would be easier for developer since the logic would be in one place?